contracts/HubPool.sol

```diff
@@ -405,6 +421,37 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
                 .rawValue;
     }
 
+    function _executeRelayerRefundOnTargetChain(MerkleLib.PoolRebalance memory poolRebalance, uint256 refundRequestId)
```

Comment:
nit: would it be clearer just to take in the chainId if that's all we're using in the poolRebalance struct?

Reply:
we also using the `refundRequest.destinationDistributionRoot` so we could pass in the `chainId` and the `destinationDistributionRoot`. I'll see if there is a gas difference, else I think it's cleaner to pass around the whole object as it makes the interfaces the same between the two methods.

Reply:
@chrismaree doesn't destinationDistributionRoot come from somewhere else (not poolRebalance)?

Reply:
yes, it comes directly from the `refundRequest` object on this contract via `refundRequest.destinationDistributionRoot`. The only thing from `MerkleLib.PoolRebalance memory poolRebalance` that is used is in this method is the `poolRebalance.chainId`. For this method, it would be cleaner to just pass in the chainId but the downside is that it then has a diffrent interface to `_sendTokensToTargetChain` which needs: `chainId`,`netSendAmount[]`,`l1Token[]`.

what do you prefer? is there a gas difference here do you think? IMHO from a readability perspective passing the whole root is cleanest but curious what you think!

Reply:
Sorry, yeah, I don't know how this is handled gas-wise. My stylistic preference is always to pass in the _simplest_ data possible, so the reader can clearly see what's being used. But this is totally up to you if it makes little or no gas difference (copying a large struct in memory could be expensive though so we might want to check on that).

Reply:
I'm going to refactor this to pass in as little data as posible. I think you right @mrice32 that this, as a general rule, is always the right way to go.